### PR TITLE
[CN-391] Update copy for School and Public Nursery flows

### PR DESCRIPTION
**Pull request by Lockyy**
### Context

https://dfedigital.atlassian.net/browse/CN-391

#### Ticket changes 

- Update copy on find-school page
- Update copy on choose-school page
- Update copy on find-childcare-provider page
- Update copy on choose childcare provider page

#### Other changes 

- Rename the display_…? methods to be descriptive
  I renamed the display_schools? and display_childcare_providers? methods to display_no_javascript_fallback_form?. I did this to make it immediately obvious what they control. That these sections of code are in place to enable to no-js fallback for selecting your institution.
- Fix copy on choose-private-childcare-provider page
- Add no-JS flow to private childcare provider form

### Guidance to review

Ticket on Jira details the required copy and has links to the prototype. Compare copy on find-school , choose-school, find-childcare-provider, and choose-childcare-provider pages to the prototypes to confirm copy is up to date.

Also, testing the no-js flows are functional in the two updated flows and the newly added section for the private childcare provider flow by moving through each interface having disabled javascript. To do this, open the dev console, hit ctrl+shift+p (windows) or cmd+shift+p (macos) and type javascript to find the `Disable Javascript` option in the dropdown.

### CODE REVIEW

File: app/lib/forms/choose_childcare_provider.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def previous_step
       :find_childcare_provider
     end
 
-    def display_childcare_providers?
+    def display_no_javascript_fallback_form?
```
Comment (peteryates) [inline app/lib/forms/choose_childcare_provider.rb:35]: This naming convention makes what's going on much clearer 👍🏽

File: app/views/registration_wizard/choose_private_childcare_provider.html.erb [hunk 2]
```diff
@@ -14,24 +18,49 @@
...
+                                 hint: { text: hint } %>
+        <% else %>
+          <%= f.govuk_radio_buttons_fieldset(:institution_identifier, legend: { size: 'xl', text: js_fallback_question, tag: "h1" }) do %>
+            <% @form.possible_institutions.each do |institution| %>
+              <%= f.govuk_radio_button :institution_identifier,
+                                       institution.identifier,
+                                       label: { text: [institution.urn, institution.name].compact.join(" - ") },
+                                       hint: { text: institution.address_string },
+                                       link_errors: true %>
+            <% end %>
```
Comment (peteryates) [inline app/views/registration_wizard/choose_private_childcare_provider.html.erb:35]: I think we just want `link_errors: true` on the first radio here:

```suggestion
            <% @form.possible_institutions.each_with_index do |institution, count| %>
              <%= f.govuk_radio_button :institution_identifier,
                                       institution.identifier,
                                       label: { text: [institution.urn, institution.name].compact.join(" - ") },
                                       hint: { text: institution.address_string },
                                       link_errors: count.zero? %>
            <% end %>
```

**Review by peteryates — COMMENTED** [review]
Looks great to me, left a couple of minor comments

File: app/views/registration_wizard/choose_private_childcare_provider.html.erb (hunk 2 above)
Comment (Lockyy) [reply app/views/registration_wizard/choose_private_childcare_provider.html.erb]: I've made that change and also updated the other templates that used the same system. :)

**Review by peteryates — APPROVED** [review]
Looks good to me.

**Pull request merged**